pkg/workflow/controller/config.go

```diff
@@ -21,12 +19,6 @@ const (
 	// ConfigFileKey is key of config file in ConfigMap
 	ConfigFileKey = "workflow-controller.json"
 
-	// GitResolverImage is key of git source resolver image in config file
```

Comment:
Also need to remove them from configmap in helm chart and release yaml.

Reply:
helm already updated, for release, need more considerations, i'd like to handle it in another PR, cyclone is cyclone, cps is cps